app/components/Navigation.js

```diff
@@ -29,59 +35,70 @@ function Navigation (props) {
         icon
       } = navigation
 
+      let item
+
       if (Component) {
-        return (
+        item = (
           <Component
             key={name}
-            as={MenuItem}
+            as={ListItem}
+            onClick={toggleDrawer}
+            button={true}
             leftIcon={
               <i className={icon} aria-hidden="true" />
             }
           />
         )
+      } else {
+        item = (
+          <NavLink to={path} key={name}>
```

Comment:
since we're touching this file, should we `h()`-ify it?

Reply:
good call :) have done!